config/config.go

```diff
@@ -107,5 +124,12 @@ func MustLoad(path string) (*Config, error) {
 	if err != nil {
 		return nil, errors.New("invalid log level")
 	}
+
+	if conf.EnableSecondaryStorage {
```

Comment:
I think we can tell whether the secondary storage is enabled or not by `SecondaryStorage`

Reply:
remove "EnableSecondaryStorage" and change to
if conf.SecondaryStorage != nil {}